main.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 The KCL Authors. All rights reserved.
+
+package main
+
+import (
+	"os"
+
+	"github.com/urfave/cli/v2"
+	command "kusionstack.io/kpm/cmds"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "kpm"
+	app.Usage = "kpm is a kcl package manager"
+	app.Version = "v0.0.1"
```

Comment:
use build version in binary: https://pkg.go.dev/runtime/debug@go1.20.2#ReadBuildInfo

Reply:
fixed.